server/RenderContext.cs

```diff
@@ -1757,5 +1806,111 @@ private void DrawMicroBorders(BorderLayer layer)
                 }
             }
         }
+
+        /// <summary>
+        /// Draw the regions
+        /// </summary>
+        /// <param name="layer">The layer</param>
+        private void DrawRegions(BorderLayer layer)
```

Comment:
I guess these _always_ fill, but only sometimes have borders. Still, can we avoid duplicating this entire method?